src/analysis/special_functions/japanese_bracket.lean

```diff
@@ -0,0 +1,214 @@
+/-
+Copyright (c) 2022 Moritz Doll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Moritz Doll
+-/
+
+import analysis.special_functions.integrals
+import analysis.special_functions.pow
+import measure_theory.integral.integral_eq_improper
+import measure_theory.integral.layercake
+import tactic.positivity
+
+/-!
+# Japanese Bracket
+
+In this file, we show that Japanese bracket $(1 + \|x\|^2)^{1/2}$ can be estimated from above
+and below by $1 + \|x\|$.
+The functions $(1 + \|x\|^2)^{-r/2}$ and $(1 + |x|)^{-r}$ are integrable provided that `r` is larger
+than the dimension.
+
+## Main statements
+
+* `integrable_one_add_norm`: the function $(1 + |x|)^{-r}$ is integrable
+* `integrable_jap` the Japanese bracket is integrable
+
+-/
+
+
+noncomputable theory
+
+open_locale big_operators nnreal filter topological_space ennreal
+
+open asymptotics filter set real measure_theory finite_dimensional
+
+variables {E : Type*} [normed_add_comm_group E]
+
+lemma jap_le_one_add_norm (x : E) : real.sqrt (1 + ∥x∥^2) ≤ 1 + ∥x∥ :=
+begin
+  have h1 : 0 ≤ 1 + ∥x∥^2 := by positivity,
+  have h2 : 0 ≤ 1 + ∥x∥ := by positivity,
+  have h3 : 0 < 2 := by positivity,
+  refine le_of_pow_le_pow 2 h2 h3 _,
+  rw [sq_sqrt h1, add_pow_two],
+  simp,
+end
+
+lemma one_add_norm_le_jap (x : E) : 1 + ∥x∥ ≤ (real.sqrt 2) * sqrt (1 + ∥x∥^2) :=
+begin
+  have h1 : 0 ≤ 1 + ∥x∥^2 := by positivity,
+  have h2 : 0 ≤ (real.sqrt 2) * real.sqrt (1 + ∥x∥^2) := by positivity,
+  have : (sqrt 2 * sqrt (1 + ∥x∥ ^ 2)) ^ 2 - (1 + ∥x∥) ^ 2 = (1 - ∥x∥) ^2 :=
+  begin
+    rw [mul_pow, sq_sqrt h1, add_pow_two, sub_pow_two],
+    norm_num,
+    ring,
+  end,
+  refine le_of_pow_le_pow 2 h2 (by norm_num) _,
+  rw [←sub_nonneg, this],
+  positivity,
+end
+
+lemma rpow_neg_jap_le_one_add_norm {r : ℝ} (x : E) (hr : 0 < r) :
+  (1 + ∥x∥^2)^(-r/2) ≤ 2^(r/2) * (1 + ∥x∥)^(-r) :=
+begin
+  have h1 : 0 ≤ (2 : ℝ) := by positivity,
+  have h2 : 0 ≤ (1 + ∥x∥^2) := by positivity,
+  have h3 : 0 < sqrt 2 := by positivity,
+  have h4 : 0 < 1 + ∥x∥ := by positivity,
+  have h5 : 0 < sqrt (1 + ∥x∥ ^ 2) := by positivity,
+  have h6 : 0 < sqrt 2 * sqrt (1 + ∥x∥^2) := mul_pos h3 h5,
+  rw [rpow_div_two_eq_sqrt _ h1, rpow_div_two_eq_sqrt _ h2, ←inv_mul_le_iff (rpow_pos_of_pos h3 _),
+    rpow_neg h4.le, rpow_neg (sqrt_nonneg _), ←mul_inv, ←mul_rpow h3.le h5.le,
+    inv_le_inv (rpow_pos_of_pos h6 _) (rpow_pos_of_pos h4 _), rpow_le_rpow_iff h4.le h6.le hr],
+  exact one_add_norm_le_jap _,
+end
+
+variables (E)
+
+lemma set_le_one_add_norm_eq_closed_ball_aux {r t : ℝ} (hr : 0 < r) (ht : 0 < t) :
+  {a : E | t ≤ (1 + ∥a∥) ^ -r} = metric.closed_ball 0 (t^(-r⁻¹) - 1) :=
+begin
+  ext,
+  simp only [norm_eq_abs, mem_set_of_eq, mem_closed_ball_zero_iff],
+  rw [le_sub_iff_add_le', neg_inv],
+  exact (real.le_rpow_inv_iff_of_neg (by positivity) ht (neg_lt_zero.mpr hr)).symm,
+end
```

Comment:
I think this is a more useful way to state this:
```suggestion
lemma le_rpow_one_add_norm_iff_mem_closed_ball {r t : ℝ} (hr : 0 < r) (ht : 0 < t) (x : E) :
  t ≤ (1 + ∥x∥) ^ -r ↔ x ∈ metric.closed_ball (0 : E) (t^(-r⁻¹) - 1) :=
begin
  simp only [norm_eq_abs, mem_closed_ball_zero_iff, le_sub_iff_add_le', neg_inv],
  exact (real.le_rpow_inv_iff_of_neg (by positivity) ht (neg_lt_zero.mpr hr)).symm,
end
```

Reply:
I changed it so that the both sides are in simp-normal form